example/basic_sample.py

```diff
@@ -63,6 +62,10 @@ def parse_arguments():
 if __name__ == '__main__':
     args = parse_arguments()
 
+    if not args.endpoint:
+        print("No Dynatrace endpoint specified, exporting to default local "
```

Comment:
The exporter init already prints this as an info log but this would not show up with the default log level, so I'm printing it to stdout here in order to make it more obvious for people just running the example.